larray/tests/test_array.py

```diff
@@ -3911,6 +3911,19 @@ def test_split_axes(self):
         assert list(res.axes.f.labels) == ['f0', 'f1']
         assert res['a0', 'b1', 'c2', 'd3', 'e2', 'f1'] == arr['a0b1', 'c2', 'd3', 'e2f1']
 
+        # labels with type 'object'
+        # =========================
+        arr = ndtest((2, 2, 2)).combine_axes(('a', 'b'))
+        fpath = abspath('test.csv')
+        arr.to_csv(fpath)
+        res = read_csv(fpath)
```

Comment:
please create the array without passing via a file. eg  `arr =  arr.set_axes([Axis(a.labels.astype(object), a.name) for a in arr.axes])`